oak_sign/src/main.rs

```diff
@@ -80,8 +71,10 @@ struct Generate {
 struct Sign {
     #[structopt(long, help = "Input PEM encoded Ed25519 private key file path")]
     pub private_key: String,
-    #[structopt(long, help = "Input file path")]
-    pub input_file: String,
+    #[structopt(long, conflicts_with = "input_string", help = "Input file path")]
+    pub input_file: Option<String>,
+    #[structopt(long, conflicts_with = "input_file", help = "Input string")]
```

Comment:
Perhaps add a bit more detail in the help to make it clear how it would be used?